Cargo.toml

```diff
@@ -15,7 +15,7 @@ keywords = ["serial", "hardware", "system", "RS232"]
 categories = ["hardware-support"]
 
 [target."cfg(unix)".dependencies]
-bitflags = ">=1.3.1, <2.1.0"
+bitflags = "1.3.1, 2.0.0"
```

Comment:
```suggestion
bitflags = "2.4.0"
```

Reply:
The current status does not compile, though.
Just `bitflags = "2.4.0"` should work for everybody. Its MSRV is also 1.56.0.

Reply:
Updated, thanks.